src/index.scss

```diff
@@ -1,54 +1,52 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter&family=Philosopher&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Philosopher:wght@700&display=swap");
 
 :root {
-  font-size: 16px;
+	font-size: 16px;
```

Comment:
please convert to using 2 spaces instead of 4

Reply:
that's really weird.. It is set to 2 spaces in my settings, moreover it looks with 2 spaces everywhere in my VSCode. But as soon as I push, here it appears with 4 spaces... and only in this pr.
Any ideas?))